src/main/kotlin/step4/racingcar/view/ResultView.kt

```diff
@@ -1,4 +1,7 @@
-package step4.racingcar.racingcar
+package step4.racingcar.view
+
+import step4.racingcar.domain.Car
+import step4.racingcar.domain.CarRacing
 
 object ResultView {
     fun view(gameResult: CarRacing) {
```

Comment:
```suggestion
    fun view(gameResult: GameResult) {
```
`CarRacing`의 전체가 아닌, 현재 필요한 `GameResult`만 넘겨주는게 좋지 않을까요?

Reply:
넵 의견주신대로 필요한 `GameResult` 만 넘기는게 좋을것 같네요~